test/parallel/test-async-wrap-check-providers.js

```diff
@@ -75,11 +75,11 @@ process.on('SIGINT', () => process.exit());
 
 // Run from closed net server above.
 function checkTLS() {
-  let options = {
+  const options = {
     key: fs.readFileSync(common.fixturesDir + '/keys/ec-key.pem'),
     cert: fs.readFileSync(common.fixturesDir + '/keys/ec-cert.pem')
   };
-  let server = tls.createServer(options, noop).listen(common.PORT, function() {
+  const server = tls.createServer(options).listen(common.PORT, function() {
```

Comment:
Well, I'd be careful nonetheless. Some parts (which I can't rembember) of the API do behave different when you pass `undefined` vs a noop. I'd rather leave that there unless we can be absolutely sure.